notebooks/book2/11/mcmc_traceplots_unigauss.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.                samples[param] = states.position[param]
</pre><p>for <code>dims</code> &gt; 1, you considered burn-in, but not here.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/908/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Ohh, thanks for pointing out, I've fixed it.</p>